Mlem/Views/Tabs/Feeds/Components/NoPostsView.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  NoPostsView.swift
+//  Mlem
+//
+//  Created by Sjmarf on 10/10/2023.
+//
+
+import SwiftUI
+
+struct NoPostsView: View {
+    @EnvironmentObject var postTracker: PostTracker
+    
+    @Binding var isLoading: Bool
+    @Binding var postSortType: PostSortType
+    @Binding var showReadPosts: Bool
+    
+    var body: some View {
+        VStack {
+            if !isLoading {
+                VStack(alignment: .center, spacing: AppConstants.postAndCommentSpacing) {
+                    
+                    let unreadItems = postTracker.hiddenItems[.read, default: 0]
+                    
+                    Image(systemName: Icons.noPosts)
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                        .frame(width: unreadItems == 0 ? 35 : 50)
+                        .padding(.bottom, unreadItems == 0 ? 8: 12)
+                    Text(title)
+                    
+                    if unreadItems != 0 {
+                        Text(
+                            (unreadItems == 1 ? "1 available post was" : "\(unreadItems) available posts were")
+                            + " hidden because you have 'hide read' enabled."
+                        )
```

Comment:
Could be more concise:

`\(unreadItems) read post \(unreadItems == 1 ? " has" : "s have") been hidden`

Along with the "show read," I think it's clear enough what's going on